Makefile

```diff
@@ -128,6 +128,17 @@ check-yml: $(TOOLING) ## Check the YAML files
 		-c .yamllint.yml \
 		.
 
+.PHONY: reproducible-build
+reproducible-build: build ## Check if the container is reproducible
+	@TAG=a ENGINE_OPTIONS=--no-cache make build
+	@TAG=b ENGINE_OPTIONS=--no-cache make build
+	@diffoci diff --semantic \
+		docker://$(IMAGE_NAME):a \
+		docker://$(IMAGE_NAME):b
+	@$(ENGINE) rmi --force \
+		$(IMAGE_NAME):a \
+		$(IMAGE_NAME):b
```

Comment:
This removal breaks subsequent runs because `.tmp/images/a` and `.tmp/images/b` still exists. I think it makes sense to remove these tags, so I'd be in favor of manually removing those files so that `make` rebuilds the tags on the next invocation.

Reply:
Resolved with 79925dbdb89e7fde887a9646cb2d2d0312e6106e